docs/howto/enumerations.md

```diff
@@ -0,0 +1,100 @@
+# How to specify enumerations
+
+Enumerations are **finite** sets of values, which may be **named** and may
+have an associated **semantic meaning**.
+
+**Note**
+While enumerations may be used to define [classes](/definitions/classes) or [labels](/definitions/labels), they don't mecessarily share the same meaning, check the definitions.
+
+Although often being of `dataType` [`sc:Text`](http://schema.org/Text), enumerations can also be of other `dataType`s (e.g. `sc:Integer`).
+
+For example, the titanic dataset uses an enumeration to specify
```

Comment:
*Maybe*, I'd put this example before the note (right after the definition).

If we leave it here, maybe change the "For example, ..." starting -- as the previous paragraph ends talking about non-text enumerations :)